code/modules/reagents/chemistry_reactions/medical.dm

```diff
@@ -151,3 +156,8 @@
 	name = "Hypervene"
 	results = list(/datum/reagent/medicine/hypervene = 3)
 	required_reagents = list(/datum/reagent/medicine/arithrazine = 1, /datum/reagent/medicine/dylovene = 1, /datum/reagent/medicine/ethylredoxrazine = 1)
+
+	/datum/chemical_reaction/roulettium
```

Comment:
```suggestion
/datum/chemical_reaction/roulettium
```

Reply:
SONOVA WHITESPACE ERROR ALWASY GETS ME AAAAAAAAAA

thank you.